framework_tvb/tvb/tests/framework/adapters/analyzers/bct_test.py

```diff
@@ -6,7 +6,7 @@
 # TheVirtualBrain-Scientific Package (for simulators). See content of the
```

Comment:
Header under some tests in framework seems to also be wrong (mentioning "scientific ... simulator" instead of "...framework...". Could you please adjust these headers also?

Reply:
I fixed those.